cli/eventstore/eventstore.spec.js

```diff
@@ -0,0 +1,45 @@
+
+const { 
+    createEventStore, 
+    syncEventStore, 
+    writeEvent 
+} = require('./eventstore')
+
+var ethUtils = require('ethereumjs-util')
+
+contract('Transmute CLI', (accounts) => {
+    let account0 = accounts[0]
```

Comment:
its a test, i don't think this matters at all. Ideally it would have a more descriptive name.